LICENSE-CODE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright 2023 Discord
+Copyright 2024 Discord
```

Comment:
```suggestion
Copyright 2016-Present Discord
```

Reply:
@Misha-133 if you want to make this change, we can close this once and for all 🥳